src/main/java/dev/thource/runelite/dudewheresmystuff/OverviewTabPanel.java

```diff
@@ -87,7 +87,14 @@ class OverviewTabPanel extends TabContentPanel {
     summaryOverview =
         new OverviewItemPanel(itemManager, null, () -> false, ItemID.NOTES, 1, LOGGED_OUT_SUMMARY);
     add(summaryOverview);
+
+    add(Box.createVerticalStrut(8));
+
+    exportItemsButton = new OverviewItemPanel(itemManager, () -> storageManagerManager.exportItems(), () -> true, ItemID.HOSIDIUS_BLUEPRINTS, 1, EXPORT_ITEMS_TEXT);
+    exportItemsButton.setVisible(false);
+    add(exportItemsButton);
```

Comment:
Instead of this having its own panel, can you make it an option in the right click menu instead please? It'll make it a bit hidden away for now, but I intend to add an `info` tab of sorts which will provide the user with some useful tips about the plugin, like that they can right click storages to reset them, etc.

See OverviewTabPanel.java:203

Reply:
Done